templates/tutorialv2/includes/tags_authors.part.html

```diff
@@ -71,7 +71,7 @@
     </span>
     <ul>
         {% for category in content.subcategory.all %}
-            {% if forloop.first %}{% elif forloop.last %} {% trans "et" %}{% else %},{% endif %} <a href="{% url 'publication:list' %}?category={{ category.slug }}">{{ category.title }}</a>
+            {% if forloop.first %}{% elif forloop.last %} {% trans "et" %}{% else %},{% endif %} <a href="{% url 'publication:list' %}?subcategory={{ category.slug }}">{{ category.title }}</a>
```

Comment:
pas si sûr motet-a, le sens de cette conditiion est plutôt

`{% if forloop.last and not forloop.first %}`